handler/consumer/json/src/test/java/io/knotx/fragments/handler/consumer/json/JsonFragmentsHandlerConsumerFactoryTest.java

```diff
@@ -48,43 +52,31 @@
   private static final String USER_KEY = "user";
   private static final String KNOTX_FRAGMENT_KEY = "_knotx_fragment";
 
-  @Test
-  @DisplayName("Fragment body should not be modified when condition not configured")
-  void expectFragmentBodyNotModifiedWhenConditionNotConfigured() {
-    //given
-    FragmentEvent original = new FragmentEvent(
-        new Fragment(EXPECTED_FRAGMENT_TYPE, new JsonObject(), FRAGMENT_BODY_JSON));
-    FragmentEvent copy = new FragmentEvent(original.toJson());
-
-    //when
-    FragmentExecutionLogConsumer tested = new JsonFragmentsHandlerConsumerFactory()
-        .create(new JsonObject()
-            .put(CONDITION_OPTION, new JsonObject().put(HEADER_OPTION, EXPECTED_HEADER)));
-    tested.accept(new ClientRequest()
-            .setHeaders(MultiMap.caseInsensitiveMultiMap().add(EXPECTED_HEADER, "true")),
-        ImmutableList.of(FragmentExecutionLog.newInstance(original)));
-
-    //then
-    assertEquals(copy, original);
+  private static Stream<Arguments> provideFragmentConsumerConfiguration() {
```

Comment:
This method does not provide consumer's configuration, but various fragments passed to the consumer being tested. Can we change the name of this method?

Reply:
fixed